.gitignore

```diff
@@ -6,3 +6,4 @@
 .hg*
 .idea
 coverage_html_report*
+*~
```

Comment:
huh?

Reply:
ignore anything with a "~' at the end.  this is the default "backup" file for linux/vim